src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -2376,14 +2376,11 @@ LedgerMaster::txnIDfromIndex(uint32_t ledgerSeq, uint32_t txnIndex)
     if (!lgr || lgr->txs.empty())
```

Comment:
Don't worry too much about adding `[FOLD]` to commit messages; the default now is to squash PRs when they get merged, so this will end up as 1 commit on `develop`, unless explicitly stated (and agreed upon) otherwise.